app/models/split.rb

```diff
@@ -82,8 +119,34 @@ def create_decision!(params = {})
     build_decision(params).tap(&:save!)
   end
 
+  def registry
+    if try(:feature_incomplete?)
+      knock_out_weightings(super)
```

Comment:
`knock_out` => `mask`? (`conceal`? `override`?) In my first reading of this I sort of assumed "knock-out" had a more specific meaning, but I'm still not quite sure what that is -- I assume it's intended to be more "to make inoperative" and less "to make unconscious"?

Reply:
Mask to inoperative is what I’m hoping knock out means. It is only exposed in effectively private API. To consumers it’s part of the presentation contract and just works